feat(profiling) add legend and explanation to the flamegraph

JonasBa opened the pull request:
![CleanShot 2023-12-06 at 13 07 24@2x](https://github.com/getsentry/sentry/assets/9317857/06413bb4-0a73-47cc-9b8c-3634398bed58)

JonasBa committed:
feat(profiling) add legend and explanation to the flamegraph

JonasBa committed:
feat(profiling): update copy

Zylphrex commented on static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx:
```diff
@@ -567,6 +569,77 @@ const DifferentialFlamegraphChangedFunctionContainer = styled('div')`
+    <DifferentialFlamegraphExplanationBarContainer>
+      <div>
+        {props.negated
+          ? `Flamegraph is showing how stack frequency will change.`
+          : `Flamegraph is showing how stack frequency has changed.`}
```
Let's make sure to wrap this in `t(...)`

JonasBa replied:
💯

JonasBa committed:
feat(profiling): wrap legend text in translation

JonasBa committed:
feat(profiling): pass fingerprint to aggregate flamegraph query (#61269)

Forward fingerprint evidence to flamegraph endpoint so it filters
profiles that include function data

---------

Co-authored-by: Tony Xiao <txiao@sentry.io>
